receiver/k8sclusterreceiver/metadata.yaml

```diff
@@ -162,6 +162,21 @@ resource_attributes:
     type: string
     enabled: false
 
+  k8s.container_runtime.version:
```

Comment:
To be a little closer to the existing [container semconv](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/container.md) I think this should be `k8s.container.runtime.version`.

Reply:
Sure